### PR TITLE
Adds $MAND

### PR DESCRIPTION
**Pull request by prtk418**
### Adding Assets

If adding a new asset, please ensure the following:
- [X] Asset is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
- [X] Add asset to bottom of `zone_assets.json`.
   - [X] `chain_name` and `base_denom` are provided and use values exactly as defined at the Chain Registry.
   - [X] `path` is provided, and the IBC channel referenced is registered at the Chain Registry (unless native to Osmosis).
   - [X] `osmosis_verified` is set to `false`
   - [X] `listing_date_time_utc` is specified and accurate
   - [ ] Optional: `transfer_methods`, `peg_mechanism`, `override_properties`, `canonical`, `categories`, where necessary (see [README](https://github.com/osmosis-labs/assetlists/tree/main?tab=readme-ov-file#how-to-add-assets) for details).

### CODE REVIEW

File: osmosis-1/osmosis-1.assetlist.json [hunk 1]
```diff
@@ -17192,6 +17192,63 @@
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/COCA.png"
       }
+    },
+    {
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-1.assetlist.json:17196]: Please do not add to this file--it will soon be deleted.

Comment (prtk418) [reply osmosis-1/osmosis-1.assetlist.json]: reverted this

File: osmosis-1/osmosis.zone_assets.json [hunk 2]
```diff
@@ -4158,6 +4158,26 @@
       "categories": ["liquid_staking"],
       "listing_date_time_utc": "2024-07-04T12:00:00Z",
       "_comment": "Stride Staked ISLM $stISLM"
+    },
+    {
+      "chain_name": "dymension",
+      "base_denom": "ibc/5A26C8DC8DA66F4DD94326E67F94510188F5F7AFE2DB3933A0C823670E56EABF",
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:4164]: oh, at the moment this asset needs to be defined at the Chain Registry as well

Comment (prtk418) [reply osmosis-1/osmosis.zone_assets.json:4164]: created a PR for the same - https://github.com/cosmos/chain-registry/pull/4957

**Pull request merged**